<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageStocksList">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_6">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_4">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_5">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageTradeInfo">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>5</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QToolButton" name="toolButtonRealTime">
                <property name="text">
                 <string>Real Time</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton5Mins">
                <property name="text">
                 <string>5MINS</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton15Mins">
                <property name="text">
                 <string>15MINS</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton30Mins">
                <property name="text">
                 <string>30MINS</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton60Mins">
                <property name="text">
                 <string>60MINS</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton1Day">
                <property name="text">
                 <string>1Day</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton1Week">
                <property name="text">
                 <string>1Week</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_9">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTabWidget" name="tabWidget">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="tabPosition">
                <enum>QTabWidget::West</enum>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="tabRealTime">
                <attribute name="title">
                 <string>Real Time</string>
                </attribute>
               </widget>
               <widget class="CandlesticksView" name="tabCandlestick">
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::CustomContextMenu</enum>
                </property>
                <attribute name="title">
                 <string>Candlestick</string>
                </attribute>
               </widget>
              </widget>
              <widget class="QStackedWidget" name="stackedWidgetExtraInfo">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page_3">
                <layout class="QVBoxLayout" name="verticalLayout_2"/>
               </widget>
               <widget class="QWidget" name="page_4">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QTabWidget" name="tabWidget_StockClasses">
                   <property name="tabPosition">
                    <enum>QTabWidget::South</enum>
                   </property>
                   <widget class="QWidget" name="tab_3">
                    <attribute name="title">
                     <string>Tab 1</string>
                    </attribute>
                   </widget>
                   <widget class="QWidget" name="tab_4">
                    <attribute name="title">
                     <string>Tab 2</string>
                    </attribute>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pushButtonExpand">
                <property name="text">
                 <string>Expand</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_10">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widgetRealTimeInfo" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="labelCode">
              <property name="text">
               <string notr="true">Code</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTopInfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widgetRealTimeAskInfo" native="true">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="1">
                <widget class="QLabel" name="labelAsk5">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelSell5">
                 <property name="text">
                  <string>ASK 5</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2" colspan="2">
                <widget class="QLabel" name="labelAskVol5">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelSell4">
                 <property name="text">
                  <string>ASK 4</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="labelAsk4"/>
               </item>
               <item row="1" column="2" colspan="2">
                <widget class="QLabel" name="labelAskVol4">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="labelSell3">
                 <property name="text">
                  <string>ASK 3</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="labelAsk3"/>
               </item>
               <item row="2" column="2" colspan="2">
                <widget class="QLabel" name="labelAskVol3">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelSell2">
                 <property name="text">
                  <string>ASK 2</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="labelAsk2"/>
               </item>
               <item row="3" column="2" colspan="2">
                <widget class="QLabel" name="labelAskVol2">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="labelSell1">
                 <property name="text">
                  <string>ASK 1</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLabel" name="labelAsk1"/>
               </item>
               <item row="4" column="2" colspan="2">
                <widget class="QLabel" name="labelAskVol1">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="5" column="0" colspan="4">
                <widget class="Line" name="line_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="labelBuy1">
                 <property name="text">
                  <string>BID 1</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLabel" name="labelBID1"/>
               </item>
               <item row="6" column="2" colspan="2">
                <widget class="QLabel" name="labelBIDVol1">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="labelBuy2">
                 <property name="text">
                  <string>BID 2</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QLabel" name="labelBID2"/>
               </item>
               <item row="7" column="2" colspan="2">
                <widget class="QLabel" name="labelBIDVol2">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="labelBuy3">
                 <property name="text">
                  <string>BID 3</string>
                 </property>
                </widget>
               </item>
               <item row="8" column="1">
                <widget class="QLabel" name="labelBID3"/>
               </item>
               <item row="8" column="2" colspan="2">
                <widget class="QLabel" name="labelBIDVol3">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QLabel" name="labelBuy4">
                 <property name="text">
                  <string>BID 4</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="1">
                <widget class="QLabel" name="labelBID4"/>
               </item>
               <item row="9" column="2" colspan="2">
                <widget class="QLabel" name="labelBIDVol4">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QLabel" name="labelBuy5">
                 <property name="text">
                  <string>BID 5</string>
                 </property>
                </widget>
               </item>
               <item row="10" column="1">
                <widget class="QLabel" name="labelBID5"/>
               </item>
               <item row="10" column="2" colspan="2">
                <widget class="QLabel" name="labelBIDVol5">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="11" column="0" colspan="4">
                <widget class="Line" name="line_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="12" column="0">
                <widget class="QLabel" name="labelCurPrice1">
                 <property name="text">
                  <string>Price:</string>
                 </property>
                </widget>
               </item>
               <item row="12" column="1">
                <widget class="QLabel" name="labelCurPrice">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="12" column="2">
                <widget class="QLabel" name="labelOpen1">
                 <property name="text">
                  <string>Open:</string>
                 </property>
                </widget>
               </item>
               <item row="12" column="3">
                <widget class="QLabel" name="labelOpen">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="13" column="0">
                <widget class="QLabel" name="labelChange1">
                 <property name="text">
                  <string>Change:</string>
                 </property>
                </widget>
               </item>
               <item row="13" column="1">
                <widget class="QLabel" name="labelChange">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="13" column="2">
                <widget class="QLabel" name="labelHigh1">
                 <property name="text">
                  <string>High:</string>
                 </property>
                </widget>
               </item>
               <item row="13" column="3">
                <widget class="QLabel" name="labelHigh">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="14" column="0">
                <widget class="QLabel" name="labelChangePercent1">
                 <property name="text">
                  <string>%Chg:</string>
                 </property>
                </widget>
               </item>
               <item row="14" column="1">
                <widget class="QLabel" name="labelChangePercent">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="14" column="2">
                <widget class="QLabel" name="labelLow1">
                 <property name="text">
                  <string>Low:</string>
                 </property>
                </widget>
               </item>
               <item row="14" column="3">
                <widget class="QLabel" name="labelLow">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="15" column="0" colspan="4">
                <widget class="Line" name="line_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widgetVolInfo" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>273</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_System">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_System"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CandlesticksView</class>
   <extends>QWidget</extends>
   <header location="global">candlesticksview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
